src/components/modules/epics/ContributionsEpicReminder.tsx

```diff
@@ -259,8 +259,8 @@ export const ContributionsEpicReminder: React.FC<Props> = ({
 
                 {!isSuccessState && (
                     <p css={formTextStyles}>
-                        We will use this to send you a single email {reminderDateWithPreposition}.
-                        To find out what personal data we collect and how we use it, please visit
+                        We will send you a maximum of two emails {reminderDateWithPreposition}. To
+                        find out what personal data we collect and how we use it, view our e visit
```

Comment:
Typo, `please` has been replaced with `view our e`

Reply:
Oh gosh - that's a bad one! Good spot 👍